onedal/tests/utils/_device_selection.py

```diff
@@ -61,20 +63,17 @@ def get_memory_usm():
         return []
 
 
-def is_dpctl_available(targets=None):
-    try:
+def is_dpctl_device_available(targets):
+    if dpctl_available:
```

Comment:
if dpctl_available check can be made an import-time check, rather than runtime.

Reply:
moving out this boolean check doesn't give a lot, but moving out package import make sense. Going to move them out from primitives.